frontend/src/metabase-lib/lib/metadata/Field.ts

```diff
@@ -34,6 +36,9 @@ import { Field as FieldRef } from "metabase-types/types/Query";
 import { FieldDimension } from "../Dimension";
 import Table from "./Table";
 import Base from "./Base";
+
+export const LONG_TEXT_MIN = 80;
```

Comment:
This file seems like a good home for this constant, but not entirely sure if there's a more sensible place for it.